.github/openapi/typescript-config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "generatorName": "typescript-angular",
+  "npmName": "@scicat-sdk/typescript-angular",
+  "ngVersion": "16.2.12",
```

Comment:
Should the ngVersion match the NODE version we use in the builder?

Reply:
ngVersion is the angular version of the project

Reply:
The ngVersion should be consistent with w/e angular version used for the scicat frontend